### PR TITLE
tests: sweep extended tests for referenced to origin

### PR DESCRIPTION
**Pull request by mfojtik**
Following @damemi changes and trying to plumb the template instance...

x-ref: https://github.com/openshift/origin/pull/23127

### CODE REVIEW

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -297,20 +296,23 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			is, err := oc.ImageClient().ImageV1().ImageStreams(oc.Namespace()).Get(name, metav1.GetOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred())
 			o.Expect(is.Status.DockerImageRepository).NotTo(o.BeEmpty())
-			directTag, ok := imageutil.StatusHasTag(is, "direct")
-			o.Expect(ok).To(o.BeTrue())
-			o.Expect(directTag.Items).NotTo(o.BeEmpty())
```
Comment (deads2k) [inline test/extended/deployments/deployments.go]: having the tag and having items is different.

File: test/extended/idling/idling.go [hunk 2]
```diff
@@ -364,14 +364,14 @@ var _ = g.Describe("idling and unidling", func() {
 
 			connWG.Wait()
 
-			g.By(fmt.Sprintf("Expecting all but %v of those connections to fail", unidlingproxy.MaxHeldConnections))
+			g.By(fmt.Sprintf("Expecting all but %v of those connections to fail", 16))
```
Comment (deads2k) [inline test/extended/idling/idling.go]: name a local constant so we know roughly what it is?

File: test/extended/templates/templateinstance_security.go [hunk 3]
```diff
@@ -255,9 +267,18 @@ var _ = g.Describe("[Conformance][templates] templateinstance security tests", f
 					},
 				}
 
+				testScheme := runtime.NewScheme()
+				utilruntime.Must(routev1.Install(testScheme))
+				utilruntime.Must(authorizationv1.Install(testScheme))
+				utilruntime.Must(storagev1.AddToScheme(testScheme))
+
+				testCodecFactory := serializer.NewCodecFactory(testScheme)
```
Comment (deads2k) [inline test/extended/templates/templateinstance_security.go]: `apitesting.SchemeForInstallOrDie`

File: test/extended/templates/templateservicebroker_bind.go [hunk 4]
```diff
@@ -59,7 +58,7 @@ var _ = g.Describe("[Conformance][templates] templateservicebroker bind test", f
 				},
 				Subjects: []corev1.ObjectReference{
 					{
-						Kind: authorizationapi.GroupKind,
```
Comment (deads2k) [inline test/extended/templates/templateservicebroker_bind.go]: promote this constant to openshift/api

Comment (mfojtik) [reply test/extended/templates/templateservicebroker_bind.go]: https://github.com/openshift/api/pull/351 will pick together with library-go

**Comment by mfojtik** [comment]
@deads2k ready (switched to openshift/api consts)

File: test/extended/templates/templateservicebroker_bind.go (hunk 4 above)
Comment (mfojtik) [reply test/extended/templates/templateservicebroker_bind.go]: picked, fixed.

**Pull request merged**